docs/ONNX_Runtime_Perf_Tuning.md

```diff
@@ -0,0 +1,94 @@
+# How to tune ONNX Runtime Performance?
+
+## Why do we need to tune performance?
+ONNX Runtime is designed to be open and scalable, it created the concept of "Execution Provider" to represents different execution kernels. 
+
+ONNX Runtime right now supports 4 CPU execution providers, which are, default(MLAS), MKL-ML, MKLDNN and nGraph. For nVidia GPU, we support CUDA and TensorRT execution providers.
```

Comment:
MKL-ML is not a provider.

Reply:
MLAS is not either, but our official releases are MLAS and MKL-ML. It will be very confusing to separate those out. That's why I added example for MKLDNN and nGraph to show how to append execution provider.

Reply:
Added a side note: (Technically, MKLML is not an execution provider since it can only enabled by using build options.)